examples/addons/http-trailers.py

```diff
@@ -15,14 +15,41 @@ def request(flow: http.HTTPFlow):
     if flow.request.trailers:
         print("HTTP Trailers detected! Request contains:", flow.request.trailers)
 
+    print(repr(flow.request.http_version))
+    if flow.request.path == "/inject_trailers":
+        if flow.request.is_http10:
+            # HTTP/1.0 doesn't support trailers
+            return
+        elif flow.request.is_http11:
+            if not flow.request.content:
```

Comment:
A lot of servers really dislike an unexpected body on GET requests so I added this check to avoid that. Also, a 0 byte chunked encoded body with trailers is legal, but odd. This ends up blocking those too. If someone wants to allow those types of requests they can tweak this script manually.